src/courseware/course/sidebar/sidebars/course-outline/components/CompletionIcon.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import { CheckCircle as CheckCircleIcon } from '@openedx/paragon/icons';
+
+import { CompletionSolidIcon, DashedCircleIcon } from '../icons';
+
+const CompletionIcon = ({ completionStat }) => {
+  const { completed, total } = completionStat ?? {};
+  const percentage = Math.min((completed / total) * 100, 100);
```

Comment:
```suggestion
const CompletionIcon = ({ completionStat: { completed = 0, total = 0 } }) => {
  const percentage = total !== 0 ? Math.min((completed / total) * 100, 100) : 0;
```
Ensure that total is not zero to avoid division by zero error

Reply:
Changed